### DISCUSSION TITLE
REST Routes & Http Verbs

### DISCUSSION THREAD

**Discussion by idigit-it** [PostgREST/postgrest#2379] [Q&A] [answered]
Hi,

I need to use PostREST for a CRUD on my postgres db.

For example I have a table "users" and I need to have an api interface like below:

GET /users
GET /users/{id}
POST /users 
PUT /users/{id}
DELETE /users/{id}

Please can I know if it is possible?

Thank you and regards,
P.

**Comment by laurenceisla** [comment, marked as answer]
Hi, right now it's not possible to modify the url, you'll need your web server to do that. There's an example in the docs using Nginx: https://postgrest.org/en/latest/admin.html#alternate-url-structure

**Reply by idigit-it** [reply]
Thanks a lot for quickly reply.
P.
